docs/platform/concepts/byoc.md

```diff
@@ -21,11 +22,26 @@ project, or organization has specific requirements. With BYOC, your Aiven
 organization gets connected with your cloud provider account by creating _custom
 clouds_ in your Aiven organization.
 
+## How it works
+
 A custom cloud is a secure environment within your cloud provider account to run
 Aiven-managed data services. By enabling BYOC, creating custom clouds, and
 setting up Aiven services within the custom clouds, you can manage your
 infrastructure on the Aiven platform while keeping your data in your own cloud.
 
+<img src={byocHowItWorks} className="centered" alt="How BYOC works" width="100%" />
```

Comment:
This image is repeating the steps below. We must avoid repetition, so I suggest you get rid of the image or get rid of the steps. 
For maintenance sake, it makes sense to get rid of the image but if you wanted to have the image to make the page more visually engaging, we could rework the image to avoid repetition and think of way to complement the content of the page.

Reply:
@jeremy-heller-aiven fyi

@ArthurFlag , to avoid repeating the same stuff, I made the image textual content leaner and moved it to the steps below the image, where I added some more useful info. Now the image includes short steps summaries only, and the steps list below includes corresponding step descriptions and links. Hope that make sense?